Pocket Casts App Clip/NowPlayingView.swift

```diff
@@ -155,6 +157,28 @@ struct NowPlayingView: View {
             }
         })
     }
+
+    private func scheduleNotification() {
+        let notificationCenter = UNUserNotificationCenter.current()
+        Task {
+            do {
+                let granted = try await notificationCenter.requestAuthorization(options: [.alert, .sound, .badge])
+                guard granted else {
+                    let settings = await notificationCenter.notificationSettings()
+                    FileLog.shared.addMessage("App Clip: Notification status: \(settings.authorizationStatus)")
+                    return
+                }
+                let content = UNMutableNotificationContent()
+                content.title = L10n.notificationsAppClipTitle
+                content.body = L10n.notificationsAppClipBody
+                let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 4.hours, repeats: false)
```

Comment:
Change this timeInterval to a something like 10.seconds to see the notification quickly